rules/Php80/Rector/NotIdentical/StrContainsRector.php

```diff
@@ -82,6 +85,13 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
+        if (isset($funcCall->args[2])) {
+            if ($this->isName($funcCall->name, 'strpos') && $this->isPositiveInteger($funcCall->args[2]->value)) {
```

Comment:
I guess the same is true for `stripos`?

Reply:
This rule only works with `strpos` y `strstr`